### ISSUE TITLE
Documentation improvements

### ISSUE THREAD

**Issue by asinghvi17** [JuliaGeometry/GeometryTypes.jl#194]
- Clarify what `Hyper` means (just signifying generalizability to $n$-dimensions)
- Simpler examples with convenience functions (Rect, Sphere, Circle, ...)

**Comment by mileslucas** [comment]
I agree a lot. I have been here a few times and looked through the documentation to see if things like triangles are implemented and I see nothing so I turn away. It wasn't until recently I actually dove into the source code and saw the over 100 exports and was truly astounded that so little is documented in the documentation. It turns out this code is _extremely useful_ for me but I just never knew it was usable. Now, too, I have to sift through source code to find the functionality I want instead of just seeing it in the docs.

**Comment by Cyrannosaurus** [comment]
I can't even find out what a Vec2f is

**Comment by Cyrannosaurus** [comment]
The documentation is laughable. What is a point? Docs don't know. A vector? Well, I guess it exists, but oh, you want to know how to go about using it? No way buddy, figure it out yourself.
![image](https://github.com/JuliaGeometry/GeometryTypes.jl/assets/70718730/ae32d496-0d6a-48c1-8ca9-7162b7460f1b)

**Comment by juliohm** [comment]
I don't think GeometryTypes.jl is maintained anymore. Take a look at Meshes.jl if you are doing geometric processing or GeometryBasics.jl if you are simply trying to visualize basic geometries.
